src/wallet/wallet.cpp

```diff
@@ -81,13 +81,33 @@ std::shared_ptr<CWallet> GetWallet(const std::string& name)
     return nullptr;
 }
 
+static std::mutex g_wallet_release_mutex;
+static std::condition_variable g_wallet_release_cond;
```

Comment:
It seems more common in bitcoin code to abbreviate condition variables as `cv` instead of `cond` and name locks `lock` instead of `lk`.